apps/urbit-bitcoin-connector/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  web:
+    image: matwet/urbit-btc-node:latest
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 55555:50002
+      - 9090:9090
+    environment:
+      ELECTRUM_IP: $ELECTRUM_IP
+      ELECTRUM_PORT: $ELECTRUM_PORT
+      BITCOIN_IP: $BITCOIN_IP
+      BITCOIN_RPC_PORT: $BITCOIN_RPC_PORT
+      BITCOIN_RPC_USER: $BITCOIN_RPC_USER
+      BITCOIN_RPC_PASS: $BITCOIN_RPC_PASS
+      BITCOIN_RPC_AUTH: $BITCOIN_RPC_AUTH
+    extra_hosts:
+        - "host.docker.internal:host-gateway"
```

Comment:
What is this required for?

Reply:
Is this to simplify the connection between the "Urbit" and "Urbit Bitcoin Connector" apps? If so we can drop this and instead the apps can refer to each other via `$APP_URBIT_IP` (setup) and `$APP_URBIT_BITCOIN_CONNECTOR_IP` (not yet setup). This will use the IPs on the internal private Docker network.

Reply:
Yes, we were trying to figure out a static address we could put in the readme to copy/paste into the terminal witout success. An env var would be great!

Reply:
Ah ok great!

Let me work on wiring that env var up, while I'm doing that do you want to tweak the container to accept an env var to write out to the instructions?

Once we're both done this should be good to merge and tag a new Umbrel release 🚀

Reply:
No problem!